Fitness/Controllers/GymDetailViewController.swift

```diff
@@ -73,6 +74,9 @@ class GymDetailViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .white
+        
+        // MARK: - Fabric
+        Answers.logCustomEvent(withName: "Checking Gym Details", customAttributes: [:])
```

Comment:
Any reason to have the empty attributes as opposed to not having that argument?

Reply:
+1, remove the argument if you aren't gonna pass in any attributes